app/app.g.less

```diff
@@ -477,9 +478,15 @@ select {
   transform: translateX(-100%);
 }
 
-.checkboxdone-enter-active {transition: width 0.3s;}
-.checkboxdone-enter {width: 0;}
-.checkboxdone-enter-to {width: 16px;}
+.checkboxdone-enter-active {
+  transition: width 0.3s;
+}
+.checkboxdone-enter {
+  width: 0;
+}
+.checkboxdone-enter-to {
+  width: 16px;
+}
```

Comment:
can you just undo the diffs in this file? afaik there isn't anything actually materially changing in it anymore